src/sw/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
+    "module": "ES6",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "pretty": true,
+    "skipLibCheck": true,
+    "lib": [
+      "es2017",
+      "esnext.asynciterable",
+      "webworker"
+    ]
+  }
+}
```

Comment:
@aomarks should this change at all based or your new src/tsconfig.json?

Reply:
@aomarks let me know if you'd like this to change in any way.

Reply:
Make sure to add `sw` to the `exclude` section of the top-level `src/tsconfig.json`.